pom/products.page.ts

```diff
@@ -10,15 +10,20 @@ export default class ProductsPage {
 
     buttonCart = () => this.page.locator(".shopping_cart_link");
     addToCartNth = () => this.page.getByText("Add to cart");
+    buttonProductNth = () => this.page.locator(".inventory_item_img.inventory_item_img");
 
-    public async goToProducts() {
 
+    
+    public async goToProducts() {
         await this.page.goto("https://www.saucedemo.com/inventory.html");
     }
 
     public async addProductToCartAndGoToCart(index: number) {
-
         await this.addToCartNth().nth(index).click();
         await this.buttonCart().click();
     }
+
+    public async clickOnProductImage(index: number) {
+        await this.buttonProductNth().nth(index).click();
```

Comment:
Cum ai mai strecurat tu inca un click aici, asa misheleshte

Reply:
din pacate nu am ce face, e din cauza la acel index

Reply:
dar as putea sa-l mut in test.... ma uit

Reply:
deja vad ca am obiceiuri (proaste) :D

Reply:
fac curat si in rest